examples/Calibration/Calibration.ino

```diff
@@ -11,18 +11,18 @@ long lastTime = 0;
 // define an 'interrupt service routine' (ISR)
 void MeterISR() {
     // let our flow meter count the pulses
-    Meter.count();
+    Meter->count();
 }
 
 void setup() {
     // prepare serial communication
     Serial.begin(9600);
 
     // enable a call to the 'interrupt service handler' (ISR) on every rising edge at the interrupt pin
-    attachInterrupt(INT0, MeterISR, RISING);
+    Meter = new FlowMeter(digitalPinToInterrupt(2), MySensor, MeterISR, RISING);
```

Comment:
Good move towards `digitalPinToInterrupt()` for better platform independence (moving away from constants in Arduino.h).

Reply:
Thanks!